# go to target goal specified with goal name
string name
---
string result